libs/wire-api/src/Wire/API/User/Search.hs

```diff
@@ -199,3 +205,58 @@ instance FromJSON TeamContact where
         <*> o .:? "managed_by"
         <*> o .:? "saml_idp"
         <*> o .:? "role"
+
+data TeamUserSearchSortBy
+  = SortByName
+  | SortByHandle
+  | SortByEmail
+  | SortBySAMLIdp
+  | SortByManagedBy
+  | SortByRole
+  | SortByCreatedAt
+  deriving (Show, Eq, Ord)
+
+instance ToByteString TeamUserSearchSortBy where
```

Comment:
round-trip test.  (there are more, but i'll let you find those.  :)

Reply:
Added bytestring rountrip tests for all new types:
- Role
- TeamUserSearchSortOrder
- TeamUserSearchSortBy
- RoleFilter

Added aeson roundtrip tests for all new types:
- TeamContact
- SearchResult TeamContact